File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/HtmlAlert.java
package tech.zolhungaj.amqapi.servercommands.globalstate;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

@CommandType("html alert")
public record HtmlAlert(
        String title,
        String message,
        @Json(name = "storeTransactionTrigger")
        boolean isFromStoreTransactionTrigger
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/globalstate/HtmlAlert.kt
package tech.zolhungaj.amqapi.servercommands.globalstate

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("html alert")
data class HtmlAlert(
    val title: String,
    val message: String,
    @Json(name = "storeTransactionTrigger") val isFromStoreTransactionTrigger: Boolean
)
